src/target/riscv/riscv.c

```diff
@@ -3590,6 +3817,20 @@ static const struct command_registration riscv_exec_command_handlers[] = {
 		.help = "Control dcsr.ebreaku. When off, U-mode ebreak instructions "
 			"don't trap to OpenOCD. Defaults to on."
 	},
+	{
+		.name = "etrigger",
+		.handler = riscv_etrigger,
+		.mode = COMMAND_ANY,
```

Comment:
Please use `COMMAND_EXEC` here:

```suggestion
		.mode = COMMAND_EXEC,
```

Reply:
Done.